src/test/java/Homework_25.java

```diff
@@ -0,0 +1,25 @@
+import PageObject.LoginPage;
+
+import org.testng.annotations.Test;
+
+import static java.sql.DriverManager.getDriver;
+
+public class Homework_25 extends BaseTest {
+    @Test
+    public void loginWithValidCredentials() throws Throwable {
+        LoginPage login = new LoginPage(getDriver());
+        login.provideEmail("burul.satybaeva@testpro.io");
+        login.providePassword("Lulkerup7710%");
+        login.clickSubmit();
+        login.verifyLogin();
+    }
+
+    @Test
+    public void loginWithValidCredentials2() throws Throwable {
```

Comment:
Please, delete `Throwable`

Reply:
![Throwable](https://github.com/Saty-lab/A53-QA-Automation-Framework/assets/149276552/6322454e-a190-4201-9c54-49c3376eaa54)
Throwable is needed for this method @Berichicko